spec/features/help_pages_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe "Help Pages", feature: true do
+  describe "Show SSH page" do
+    before do
+      login_as :user
+    end
+    it "replace the variable $your_email with the email of the user" do
+      visit help_page_path(category: "ssh", file: "ssh.md")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.